### PR TITLE
Broke out cloud run terraform configuration doc snippets into their own files

### PR DESCRIPTION
**Pull request by thomasmburke**
Developers have reported confusion without the full context for where a terraform block should be located or wind up missing a } in their own configuration. This update will help developers where exactly certain cloud run configurations should be defined.

See comments in cl/509670152 for further details driving this change.

### CODE REVIEW

File: cloud_run_configuration_concurrency/concurrency.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/**
```
Comment (betsy-lichtenberg) [inline cloud_run_configuration_concurrency/concurrency.tf:1]: Please rename all config files `main.tf`

**Comment by betsy-lichtenberg** [comment]
Please move all new directories under the /run parent directory so that we can avoid incurring additional technical debt. See [go/terraform-docs-samples-repo-cleanup-actions](http://goto.google.com/terraform-docs-samples-repo-cleanup-actions)

**Comment by betsy-lichtenberg** [comment]
Are you removing this content from existing Cloud Run samples? I don't see any deletions in the diffs.

File: cloud_run_configuration_concurrency/concurrency.tf (hunk 1 above)
Comment (thomasmburke) [reply cloud_run_configuration_concurrency/concurrency.tf]: Done.

**Comment by thomasmburke** [comment]
> Are you removing this content from existing Cloud Run samples? I don't see any deletions in the diffs.

To avoid breakage of 10 existing docs that use `cloud_run_service_configuration/main.tf` I am not removing it in this PR. Once this PR is merged, we can update the public docs to use the new decoupled terraform files, then a new PR can created to remove `cloud_run_service_configuration/main.tf`.

File: run/cloud_run_configuration_containers/main.tf [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Example configuration of a Cloud Run service with command and args
+
+# [START cloudrun_service_configuration]
```
Comment (betsy-lichtenberg) [inline run/cloud_run_configuration_containers/main.tf:19]: The region tags have to be globally unique. Can you add more context to each region tag to make them unique?

Comment (thomasmburke) [reply run/cloud_run_configuration_containers/main.tf]: Done!

**Comment by betsy-lichtenberg** [comment]
Test failure unrelated to this PR

**Review by betsy-lichtenberg — APPROVED** [review]

**Pull request merged**